OpenSim/Common/ModelDisplayHints.h

```diff
@@ -102,6 +102,9 @@ class OSIMCOMMON_API ModelDisplayHints : public Object {
     OpenSim_DECLARE_PROPERTY(show_debug_geometry, bool,
         "Flag to indicate whether or not to show debug geometry, default to false.");
 
+    OpenSim_DECLARE_PROPERTY(display_scale_factor, double,
+        "Scale factor to apply to decorations (that have no inherent size) but are model size dependent (e.g. Marker, halfplane size defaults to 1.");
```

Comment:
If you intended to convey that the decoration for the ContactHalfSpace has length/width 1.0 by default, then I think this information is best conveyed in the documentation for ContactHalfSpace.